### PR TITLE
Replace 2.4.1 and 1.3.7 input manifest with tags

### PR DESCRIPTION
**Pull request by gaiksaya**
Signed-off-by: Sayali Gaikawad <gaiksaya@amazon.com>

### Description
Replace 2.4.1 and 1.3.7 input manifest with tags

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: manifests/1.3.7/opensearch-1.3.7.yml [hunk 1]
```diff
@@ -10,13 +10,13 @@ ci:
...
     repository: https://github.com/opensearch-project/OpenSearch.git
-    ref: db18a0d5a08b669fb900c00d81462e221f4438ee
+    ref: tags/1.3.7
     checks:
       - gradle:publish
       - gradle:properties:version
   - name: common-utils
     repository: https://github.com/opensearch-project/common-utils.git
-    ref: c350d4f243b817125b39b97de5b43fad45e8209d
+    ref: tags/1.3.7
```
Comment (zelinh) [inline manifests/1.3.7/opensearch-1.3.7.yml:19]: The tags for plugins should be 4 digits.

Comment (gaiksaya) [reply manifests/1.3.7/opensearch-1.3.7.yml]: Corrected!

File: manifests/1.3.7/opensearch-dashboards-1.3.7.yml [hunk 2]
```diff
@@ -9,35 +9,35 @@ ci:
...
-    ref: ce5db118ccb484da6778992f9175878df01cbd3d
+    ref: tags/1.3.7
   - name: functionalTestDashboards
     repository: https://github.com/opensearch-project/opensearch-dashboards-functional-test.git
-    ref: 2448285f99fc92cfdcbd483882df4cf3a968fea5
+    ref: tags/1.3.7
   - name: securityDashboards
     repository: https://github.com/opensearch-project/security-dashboards-plugin.git
-    ref: c34dc55b48c2b15c53b726091cb12f793fdb229a
+    ref: tags/1.3.7
```
Comment (zelinh) [inline manifests/1.3.7/opensearch-dashboards-1.3.7.yml:18]: Seems like miss to change this one

Comment (gaiksaya) [reply manifests/1.3.7/opensearch-dashboards-1.3.7.yml]: Fixed now. Thanks!

**Review by zelinh — APPROVED** [review]
LGTM.

**Pull request merged**